syntax = "proto3";

package math;

message SqrtRequest
{
	int32 number = 1;
}

message SqrtResponse
{
	double result = 1;
}

service SqrtService
{
	rpc Calculate(SqrtRequest) returns (SqrtResponse) {}
}

message OperandRequest
{
	int32 number = 1;
}

message ResultResponse
{
	int32 result = 1;
}

service MathService
{
	rpc CalculateTriple(OperandRequest) returns (ResultResponse) {}
	rpc CalculateDouble(OperandRequest) returns (ResultResponse) {}
}
